architecture/routing.adoc

```diff
@@ -112,10 +162,25 @@ Passthrough termination is configured by setting `tls:termination` to `passthrou
 }
 ----
 
-=== Re-encryption Termination
-Re-encryption is a type of edge termination where the client encrypts communication with a certificate, which is then re-encrypted with a different certificate when the traffic reaches the destination; with the router using health checks to determine the authenticity of the host.
+<1> Set the `*termination*` field to `passthrough`.
+====
+
+The destination, such as an *Nginx*, *Apache*, or another *HAProxy* instance, is
+then responsible for serving certificates for the traffic:
+
+[[re-encryption-termination]]
+*Re-encryption Termination*
+
+Re-encryption is a type of edge termination where the client encrypts
+communication with a certificate, which is then re-encrypted with a different
+certificate when the traffic reaches the destination. The router uses health
+checks to determine the authenticity of the host.
```

Comment:
Similar to my suggestion for a hyperlink to TLS termination; should there be one for Health Checks?